src/sample/stuDashController.java

```diff
@@ -203,7 +203,11 @@ public void displayNRIC() {
       Scanner sc = new Scanner(stuinfoFile);
       String icnumber;
       icnumber = Files.readAllLines(Paths.get("data/Student Dashboard/" + id + ".txt")).get(1);
-      nric.setText(icnumber);
+      if(icnumber.contains("empty")){
```

Comment:
I think it is no longer  with the word"empty" for the things in this file

Reply:
What word is it now?

Reply:
nothing I think

Reply:
Gg, my code is outdated, no wonder